src/config/bypass.config.ts

```diff
@@ -0,0 +1,7 @@
+import { registerAs } from '@nestjs/config';
+
+export default registerAs('bypass', () => ({
+  bypassed: process.env.EASEY_AUTH_API_BYPASS || false,
+  pass: process.env.EASEY_AUTH_BYPASS_PASS || 'password',
+  users: process.env.EASEY_AUTH_API_BYPASS_USERS || '["kherceg-dp"]',
+}));
```

Comment:
lets change this to be cdxBypass with the following properties...

enabled: process.env.EASEY_AUTH_API_CDX_BYPASS_ENABLED || false,
password: process.env.EASEY_AUTH_API_CDX_BYPASS_PASSWORD || 'IC@nn0tL0g1nIn',
users: process.env.EASEY_AUTH_API_CDX_BYPASS_USERS || [],